annotations/sort.json

```diff
@@ -15,6 +16,7 @@
         {
             "predicate": "default",
             "class": "pure",
+            "agg": "sort",
```

Comment:
I am not sure I understand why there are two aggregate fields `aggregate` at the top and `agg` in the predicate. The aggregator should only be in the predicate since it might only work for a specific invocation. We can merge for now but this should be fixed :)